sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -39,6 +40,15 @@ def test_mcd():
     # 1D data set
     launch_mcd_on_dataset(500, 1, 100, 0.001, 0.001, 350)
 
+def test_fast_mcd():
+
+    X = np.arange(100)
+    try:
+        assert_warns(DeprecationWarning, fast_mcd, X)
```

Comment:
You could also check that the `DeprectionWarning` is only due to the 1D X using `assert_warns_message`, if you don't mind ;)